Could not find proj_api.h

When I try to install postgis, I get this error:
```
==> Downloading https://download.osgeo.org/postgis/source/postgis-2.4.4.tar.gz
######################################################################## 100.0%
==> ./configure --with-projdir=/usr/local/opt/proj --with-jsondir=/usr/local/opt/json-c --with-pgconfig=/usr/local/opt/postgresql/bin/pg_config --disable-nls
Last 15 lines from /Users/sebastienkerbrat/Library/Logs/Homebrew/postgis/01.configure:
checking for gmsgfmt... /usr/local/opt/gettext/bin/msgfmt
checking for xgettext... /usr/local/opt/gettext/bin/xgettext
checking for msgmerge... /usr/local/opt/gettext/bin/msgmerge
checking for ld used by GCC... /Library/Developer/CommandLineTools/usr/bin/ld
checking if the linker (/Library/Developer/CommandLineTools/usr/bin/ld) is GNU ld... no
checking for shared library run path origin... /bin/sh: build-aux/config.rpath: No such file or directory
done
checking for CFPreferencesCopyAppValue... yes
checking for CFLocaleCopyCurrent... yes
checking whether to use NLS... no
Using user-specified proj directory: /usr/local/opt/proj
checking proj_api.h usability... no
checking proj_api.h presence... no
checking for proj_api.h... no
configure: error: could not find proj_api.h - you may need to specify the directory of a PROJ.4 installation using --with-projdir

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/cloverhealth/homebrew-tap/issues
```

I think this might be caused by this update in proj 6.0.0 that would cause it to be incompatible with postgis 4.4:

```
Deprecated the proj_api.h interface. The header file is still available but will be removed with the next major version release of PROJ. It is now required to define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H before the interface can be used (#836)
```

A workaround could be to make the script use an older version of proj but I didn't manage to do it.

--- a/README.md
+++ b/README.md
@@ -1,14 +1,14 @@
 # homebrew-tap
 
 This is Clover Health's Homebrew Tap, which features formulas to help pin
-Postgres to 9.6 and Postgis to 2.4.
+Postgres to 9.6 and Postgis to 2.5.
 
 Homebrew will pull in the latest version of formulas when they are upgraded,
 meaning that users can inadvertently be upgraded to Postgresql 10. The
 postgresql.rb formula here ensures that 9.6.10 is installed, and the postgis.rb
-formulate ensures that 2.4.4 is installed.
+formulate ensures that 2.5.2 is installed.
 
-## Installing Postgres 9.6 and Postgis 2.4
+## Installing Postgres 9.6 and Postgis 2.5
 
 First ensure you have upgraded to the latest homebrew:
 
@@ -59,7 +59,7 @@ Switch to 9.6.10 with:
 brew switch postgresql 9.6.10
 ```
 
-Postgis 2.4 can be installed with:
+Postgis 2.5 can be installed with:
 
 ```sh
 brew install cloverhealth/tap/postgis

--- a/postgis.rb
+++ b/postgis.rb
@@ -1,8 +1,8 @@
 class Postgis < Formula
   desc "Adds support for geographic objects to PostgreSQL"
   homepage "https://postgis.net/"
-  url "https://download.osgeo.org/postgis/source/postgis-2.4.4.tar.gz"
-  sha256 "0663efb589210d5048d95c817e5cf29552ec8180e16d4c6ef56c94255faca8c2"
+  url "https://download.osgeo.org/postgis/source/postgis-2.5.2.tar.gz"
+  sha256 "b6cb286c5016029d984f8c440947bf9178da72e1f6f840ed639270e1c451db5e"
   revision 1
 
   head do

